odoo/addons/base/populate/res_user.py

```diff
@@ -17,9 +17,13 @@ class Users(models.Model):
         'large': 10000,
     }
 
-    _populate_dependencies = ["res.partner"]
+    _populate_dependencies = ["res.groups", "res.partner"]
 
     def _populate_factories(self):
+        group_ids = self.env['res.groups'].search([])
```

Comment:
Why is here using `search()` instead of `self.env.registry.populated_models['res.groups']`?